### ISSUE TITLE
UI does not show required property prefixes on app info view

### ISSUE THREAD

**Issue by dturanski** [spring-cloud/spring-cloud-dataflow#3815]
**Description:**
The UI shows only simple property names, omitting prefixes that are required to configure the app in some cases. `tasklauncher-dataflow` sink is one example.
**Release versions:**
2.0.5.BUILD-SNAPSHOT

**Steps to reproduce:**
Click on the app details view for `tasklauncher-dataflow` sink and compare to `app info` in the shell.

![Screen Shot 2020-03-03 at 1 29 43 PM](https://user-images.githubusercontent.com/886859/75807307-21441480-5d53-11ea-9203-3cc52679f679.png)
![image](https://user-images.githubusercontent.com/886859/75807401-5486a380-5d53-11ea-9246-5cd055601310.png)

**Comment by sabbyanandan** [comment]
@oodamien: Is this an optimization that we did in the UI so that users won't have to repeat the prefixes? Curious where the consistency diverges (backend or ui).

cc:/ @jvalkeal

**Comment by dturanski** [comment]
The deploy screen shows only the simple property names. That's ok, but I think the App info view should show fully qualified names.

**Comment by sabbyanandan** [comment]
@oodamien / @jvalkeal: Do you have any thoughts to share?

**Comment by sabbyanandan** [comment]
@oodamien / @jvalkeal: ping

**Comment by sabbyanandan** [comment]
Likely related to https://github.com/spring-cloud/spring-cloud-dataflow/issues/4188.

**Comment by ilayaperumalg** [comment]
Yes, this is indeed the optimisation done at the UI level as the REST response provides the fully qualified names of the properties (as that is what we use at the Shell as well).
We need to aim addressing this along with #4188
